.idea/codeStyles/Project.xml

```diff
@@ -241,11 +241,8 @@
       </indentOptions>
```

Comment:
trzeba to usunąć ze zmian

Reply:
Mi cały wieczór wczoraj IntelliJ dorzucał zasady DLA PROTO, bo pobrałam jakiś plugin na cześć rozprochów, także zawsze mogło być gorzej